src/parsing/parsers/clickhouse/jison/create/create_table.jison

```diff
@@ -195,69 +195,70 @@ ColumnOption_EDIT
 ColumnDataType
  : PrimitiveType
  | ArrayType
- | MapType
- | StructType
- | ArrayType_INVALID
- | MapType_INVALID
- | StructType_INVALID
+ | LowCardinalityType
+ | EnumType
+ | 'Nullable' '(' PrimitiveType ')'
+ | 'FixedString' OptionalTypeLength
+ | 'AGGREGATE_FUNCTION' '(' 'REGULAR_IDENTIFIER' ',' PrimitiveTypeList ')'
+ | 'SIMPLE_AGGREGATE_FUNCTION' '(' 'REGULAR_IDENTIFIER' ',' PrimitiveType ')'
```

Comment:
I cant explain it, but when I call 'AGGREGATE_FUNCTION' token with PascalCase, I get parse errors, so I decided to call all his data types with upper snake case